flang/lib/Semantics/check-omp-structure.cpp

```diff
@@ -11,6 +11,7 @@
 #include "flang/Parser/parse-tree.h"
 #include "flang/Semantics/tools.h"
 #include <algorithm>
+#include <iostream>
```

Comment:
If possible, can we avoid using iostream here?

Reply:
Yes, sorry for that, its an unrelated change - I was trying to dump the attribute list to `std::cout`, sorry. Will remove it.